region/brinstar/pink.json

```diff
@@ -3194,15 +3230,54 @@
                     {"or": [
                       "SpaceJump",
                       "canWalljump",
-                      "Grapple"
+                      {"and": [
+                        "Grapple",
+                        {"obstaclesNotCleared": ["C"]}
+                      ]},
+                      {"and": [
+                        "HiJump",
+                        "canSpringBallJumpMidAir"
+                      ]}
                     ]},
                     {"or": [
                       "Wave",
                       {"obstaclesCleared": ["B"]}
                     ]}
                   ],
                   "clearsObstacles": ["B"],
-                  "note": "The reverse gate glitch may be executed earlier from node 1 -> 1."
+                  "devNote": "The reverse gate glitch may be executed earlier from node 1 -> 1."
+                },
+                {
+                  "name": "Speedy Jump Onto Grapple Block",
+                  "notable": false,
+                  "requires": [
+                    "HiJump",
+                    "canTrickyDashJump",
+                    "canDisableEquipment",
```

Comment:
```suggestion
                    "canTrickyJump",
                    "SpeedBooster",
```

Reply:
This feels easier than a trickyDashJump to me.  And the runway length sets itself up for the jump so you don't need to think about pixel positions.
Jumping from the grapple block can be made without turning anything off, and `canTrickyJump` is already required.

Reply:
When you suggested this in the past, I didn't realize you meant to still have a `canTrickyJump` on it. I think this will work fine.